electron/main/index.ts

```diff
@@ -64,6 +68,14 @@ async function createWindow() {
   });
   sdk.register('btc', Btc.createInstance);
   sdk.register('tbtc', Tbtc.createInstance);
+  sdk.register('eth', Eth.createInstance);
+  sdk.register('gteth', Gteth.createInstance);
+  sdk.register('eos', Eos.createInstance);
+  sdk.register('teos', Teos.createInstance);
+  sdk.register('xlm', Xlm.createInstance);
+  sdk.register('txlm', Txlm.createInstance);
+  sdk.register('xrp', Xrp.createInstance);
+  sdk.register('txrp', Txrp.createInstance);
```

Comment:
create a function called `createBitGoApi` that encapsulates the creation of the sdk and the registering of its instances, the return value should be `sdk` that way you're not duplicating logic here.